Code/Query/test.cpp

```diff
@@ -305,5 +306,4 @@ int main() {
   test5();
   test6();
   test7();
-  return 0;
 }
```

Comment:
I didn't see any pickle tests, might have missed them though.

Reply:
There's no code for directly serializing queries themselves, but the molecule pickling code does need to handle this.
The test that I added to `file_parsers_catch.cpp` checks that the new information is correctly pickled/unpickled.
I just added an additional test there to be sure it's doing the right thing.